apps/settings/js/panels/achievements/achievements_template.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * The template function for generating an UI element for an Addon item.
+ *
+ * @module achievements/achievements_template
+ */
+define(function(require) {
+  'use strict';
+
+  function achievementsTemplate(onClick, achievement) {
```

Comment:
When requesting an container using a template `ListView` passes an existing container to the template as the last parameter. Use the existing container to save the time of recreating elements. You can refer to this[[1]].

[1]: https://github.com/mozilla-b2g/gaia/blob/master/apps/settings/js/panels/addons/addons_template.js#L18